src/types.ts

```diff
@@ -1,4 +1,5 @@
-export type ResponsiveTableColumn = { id: string | number; text: string };
+export type ResponsiveTableColumn = { field: string; label: string };
```

Comment:
`field` può essere `string` or `number`
```suggestion
export type ResponsiveTableColumn = { field: string | number; label: string };
```

Reply:
Dato queste modifiche, direi che possiamo aggiornare anche il `README` con i type corretti

Reply:
Allora... Questo non è il type del set di dati, ma è il type della mappatura delle colonne. 
Posto che javascript forza le chiavi di un oggetto a string questo può tranquillamente rimanere stringa.

<img width="183" alt="image" src="https://user-images.githubusercontent.com/92720697/226561515-e9d2f669-a914-40dd-986e-69a1b7d053be.png">

Per quanto riguarda il set di dati usiamo già il type `string | number`.

Reply:
Ah figo non lo sapevo allora meglio `string` !